apps/app-submission-guidelines.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: page
+key: apps-submit-guide
+title: Submission guidelines
+---
+
+We're excited that you want to create an app for ePages!
+Before you start creating it, it's important that you understand our guidelines.
+Our app submission guidelines will help you steer clear of difficulties as you develop your app and provide a structure for you to submit an app that suits our needs.
+The objective of your app should be meaningful and help our customers achieve their goals. Have fun coding!
+
+# Functionality
+
+* Make sure your app performs as described and advertised.
+Apps that crash or have bugs will be rejected.
+* Be transparent with your app documentation and features.
+Apps that include undocumented or hidden features will be rejected.
+* We love to receive the final and production ready version of your app.
+We will not accept demo, trial or test versions.
+* Save your time and effort and don't duplicate an app that is already available in the Apps & Themes Store.
+We may reject it.
+* Concentrate on a reasonable and useful app.
+Apps that provide trick or mock a functionality will be rejected.
+
+# Metadata
+
+* We are keen on complete and meaningful text and descriptions in apps.
+We will not accept apps that contain placeholder text.
+* Relevance rules.
+Only provide us with apps that include names, descriptions, screenshots etc. that are relevant to the app content and functionality.
+Apps that do not comply with this will be rejected.
+* Assign appropriate keywords for your app.
+Inadequate keywords may be changed or deleted.
+* Only submit your app for review with all included URLs fully functional.
+* Make sure that your app respects privacy and does not display personal information of a real person without permission.
+
+# Privacy
+
+* Make sure your app notifies and obtains user consent before collecting, transmitting or using customer or location data.
+Apps that perform contrary to this will be rejected.
+* We set a high value on security.
+We will reject apps that transmit viruses as well as files, code or programs that disrupt, damage or spy on our customer's data.
+* Protect customer data.
+We will not accept apps that require our customers to share e.g. their email address or date of birth in order to function.
+
+# Other
+
+* Our customers are our greatest asset and we respect them.
+We will not accept apps that are defamatory, insulting or mean.
+* Apps must comply with all legal requirements in any country where they are made available to our customers.
+It is in the responsibility of the app developer to understand and conform to the respective local laws and regulations.
```

Comment:
It is in the responsibility of the app developer, to understand and conform to the respective local laws and regulations. Please send us an email to xx if you have any inquiries. 

*perhaps this is a good clause to add to save our butts.. we allow them to contact us and ask questions specific to this topic*